core/src/main/java/org/mule/runtime/core/api/streaming/bytes/factory/AbstractCursorStreamProviderFactory.java

```diff
@@ -36,7 +36,7 @@
 
   protected final ByteBufferManager bufferManager;
   protected final StreamingManager streamingManager;
-  protected final boolean trackCursorProviderClose;
+  protected static boolean trackCursorProviderClose;
```

Comment:
don't just make it static... resolve the value statically.. the original comment aims to not resolving the SP each time the factory is created